### ISSUE TITLE
Brexit.cy incorrect/missing lang attributes

### ISSUE THREAD

**Issue by danacotoran** [alphagov/collections#2628]
The [Welsh language version of the Brexit page]( https://www.gov.uk/brexit.cy) has large chunks of English language content without appropriate lang attributes which is an accessibility problem. The bits of content which are in English should be identified with `lang="en"` so that screenreader software reads them out properly.

Additionally, the links in the bottom section (a part of the page which actually is in Welsh) has `lang="en"` which means it is not read out correctly by AT (for instance on that portion of the page VoiceOver on Mac reads the character `â` as "a-circumflex" as can be heard in the recording below)

https://user-images.githubusercontent.com/7116819/146046797-9c561035-97cf-4615-9a9f-6c015074f6b3.mov

----- 

https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-other-lang-id.html

**Comment by sihugh** [comment]
Thanks for this @danacotoran. The page no longer exists, so I'm going to close the issue.
